src/main/java/ChooseAccountPage.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class ChooseAccountPage {
+
+    private final WebDriver driver;
+    private final By youAreLoggedOutText = By.xpath("//h3[@class='AuthAccountList-unathorizedAccountsTitle']");
+    private final By accountsList = By.xpath("//div/a[@class='AuthAccountListItem']");
+    private final By currentAccountName = By.xpath("//span[@class='CurrentAccount-displayName']");
+
+    public ChooseAccountPage(WebDriver driver) {
```

Comment:
No needs to pass WebDriver instance as parameter

Reply:
Constructor was changed